### PR TITLE
New: ItemExtendedCost.dbc and ItemPurchaseGroup.dbc, Updated npc_vendor#extendedcost

### PR DESCRIPTION
**Pull request by TheSCREWEDSoftware**
### Description

- Created `ItemExtendedCost.dbc` (`itemextendedcost_dbc`)
- Updated `npc_vendor#extendedcost` to have proper link to the new`itemextendedcost_dbc`
- Created `ItemPurchaseGroup.dbc` (`itempurchasegroup_dbc`)

### Related Issue

Not aware

@pangolp new changes to the EN files

### CODE REVIEW

**Review by Helias — APPROVED** [review]

File: docs/itempurchasegroup_dbc.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+{% include warning.html content="THIS IS UN-USED AND INFORMATION CAN BE WRONG" %}
+
+**The \`itempurchasegroup_dbc\` table**
+
+**Table Structure**
+
+| Field                             | Type    | Attributes | Key | Null | Default | Extra | Comment |
+| --------------------------------- | ------- | ---------- | --- | ---- | ------- | ----- | ------- |
+| [ID](#id)                         | INT     | UNSIGNED   | PRI | NO   | 0       |
+| [ItemID_1](#itemid_1)             | INT     | UNSIGNED   |     | NO   | 0       |
```
Comment (Kitzunu) [inline docs/itempurchasegroup_dbc.md:14]: samne as other pr

**Comment by TheSCREWEDSoftware** [comment]
Corrected

**Comment by TheSCREWEDSoftware** [comment]
Ready to be reviewed.

File: docs/itemextendedcost_dbc.md [hunk 2]
```diff
@@ -0,0 +1,1081 @@
...
+
+The amount of required personal arena rating.
+
+### ItemPurchaseGroup 
+
+THIS IS UN-USED
+
+ID that's used in [ItemPurchaseGroup.dbc](itempurchasegroup.dbc#id). <br>All entries use "0". No more information
+
+### DBC File Content
```
Comment (Kitzunu) [inline docs/itemextendedcost_dbc.md:100]: no

File: docs/itempurchasegroup_dbc.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+| --- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------------------------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | ------------- | -------------- |
+| 1   | 34529    | 34530    | 33006    | 0        | 0        | 0        | 0        | 0        | Vengeful Gladiator Ranged Weapon |                |                |                |                |                |                |                |                |                |                |                |                |                |                |               | 16712190       |
+
+</details>
```
Comment (Kitzunu) [inline docs/itempurchasegroup_dbc.md:154]: why

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: docs/itemextendedcost_dbc.md (hunk 2 above)
Comment (TheSCREWEDSoftware) [reply docs/itemextendedcost_dbc.md]: Done

File: docs/itempurchasegroup_dbc.md (hunk 3 above)
Comment (TheSCREWEDSoftware) [reply docs/itempurchasegroup_dbc.md:154]: Done

**Pull request merged**